krun/tests/test_util.py

```diff
@@ -124,30 +124,37 @@ def test_read_results():
 
 
 def test_dump_results():
-    config_file = 'krun/tests/example.krun'
```

Comment:
Changes in `test_dump_results` should probably suggest a change in the test that deals with reading results and also the binary results files in `krun/tests/`